### PR TITLE
This is the Desktop version of my portfolio website

### PR DESCRIPTION
**Pull request by lily-coder**
I have used media queries to create the desktop version of my portfolio website

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -10,124 +10,225 @@
...
-        <i class="fa fa-medium white"></i>
-    </div>
-    <div class="work">
+        <i class="fa fa-medium white"></i> 
+      </div>
+      <div class="work">
         <i class="fa fa-arrow-down"></i>
         <p>Work</p>
+      </div>
     </div>
```
Comment (rloterh) [inline index.html:73]: - [ ] The _layout_ of contents, _spacing_, and _fonts_ of your design does not match the provided [Figma template](https://www.figma.com/proto/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A976&scaling=min-zoom&page-id=23%3A10) for both mobile and desktop versions:
- The _font sizes_ and _font-weight_ and _spacing (margins and paddings)_ do not match the original design.
- The font family of the `h1` element does not match the design of the provided template.
- Your page's width does not fit on the media screen and therefore it's overlapping on the desktop version.
Kindly note that we actually want our designs to close look like the provided template above as much as possible. 

**Your designs:**
image

image

image

**The original designs:**
image

image

Please compare both designs closely and let's improve on the enumerated issues above.😉🛠💻🍷

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Lilian 

Good job so far on this project!👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there!👍 

## Good Points 👍  
- No linter errors

## Required Changes ♻️
- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by lily-coder** [comment]
I have made changes as requested by the reviewer as follows;
-I have used desktop breakpoints value for min-width property in the range 992px-1100px
-I have edited the layout of contents, spacing, and fonts of my design to match the figma template both for mobile and desktop version
-I have used font-sizes and font-weights and spacing as the original template
- The page width is now fitting the media screen

File: style.css [hunk 2]
```diff
@@ -184,50 +336,666 @@ a {
...
+    width: 100px;
+    height: 50px;
+    left: 830px;
+    top: 3200px;
+    background: #edecff url(.assets/img/ellipse-2.svg);
+    filter: blur(40px);
+  }
+
+  footer {
+    display: flex;
```
Comment (Lameck1) [inline style.css:998]: - [ ] [OPTIONAL] Let's strive to keep the code DRY. This property:
```CSS
display: flex
```
has been used several times in the CSS. Please refactor it to a class and apply the class in the HTML file to elements that share the same display property.

### Example code
```CSS
.d-flex {
  display: flex;
  align-items: center;
}
```

Then apply this class to HTML element that needs this behavior:

```HTML
  <section class="d-flex">
```

**Review by Lameck1 — APPROVED** [review]
Hi, @lily-coder,

Your project is complete!  It's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**